backend/app/Services/Webhooks/WebhookDispatcher.php

```
@@ -0,0 +1,67 @@
+            'event' => $event,
+            'data' => $data,
+            'timestamp' => now()->toIso8601String(),
+        ];
+
+        foreach ($webhooks as $webhook) {
+            if (! $webhook->subscribesTo($event)) {
+                continue;
+            }
+
+            try {
+                $delivery = WebhookDelivery::create([
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\WebhookDelivery::create().


(staticMethod.notFound)